### ISSUE TITLE
Multiple pipeline support

### ISSUE THREAD

**Issue by clwluvw** [SpencerMalone/logstash-output-prometheus#10]
Hi. In multiple pipeline both of prometheus outputs bind in a single pid so one of them isn't working!

**Comment by clwluvw** [comment]
@SpencerMalone Can you please help to solve this? I can't even have two prometheus on one pipeline too in logstash 7.7.1!

**Comment by SpencerMalone** [comment]
Are you specifying multiple ports?

**Comment by clwluvw** [comment]
@SpencerMalone Yes but another port isn't responding anymore! Can you test it on your own?

**Comment by abbaspw21** [comment]
@SpencerMalone @clwluvw  Is there any fix for this
